common/libs/validators.js

```diff
@@ -83,3 +84,12 @@ export function isPositiveIntegerOrZero(number: number): boolean {
   }
   return number >= 0 && parseInt(number) === number;
 }
+
+export function isValidABIJson(str) {
```

Comment:
I had originally written this when I was making generic inputs for all sorts of things. I had since taken that out, but this was left behind. I'll remove it.